web/client/plugins/Login.jsx

```diff
@@ -51,96 +55,213 @@ import { isAdminUserSelector } from '../selectors/security';
  * }
  * ```
  *
-  * @class Login
+  * @class LoginPlugin
   * @memberof plugins
   * @static
+  * @name LoginPlugin
   *
   * @prop {string} cfg.id identifier of the Plugin, by default `"mapstore-login-menu"`
-  * @prop {object} cfg.menuStyle inline style for the menu, by default:
-  * @prop {object} cfg.isUsingLDAP flag refers to if the user with type LDAP or not to manage show/hide change psasword, by default: false
-  * ```
-  * menuStyle: {
-  *      zIndex: 30
-  * }
-  *```
+  * @prop {boolean} cfg.isUsingLDAP flag refers to if the user with type LDAP or not to manage show/hide change psasword, by default: false
+  * @prop {object} cfg.user the user object
+  * @prop {object[]} cfg.entries the entries of the menu
+  * @prop {boolean} cfg.enableRulesManager flag to enable/disable the rules manager
+  * @prop {boolean} cfg.enableImporter flag to enable/disable the importer
+  * @prop {function} cfg.onItemSelected function to call when an item is selected
+  * @prop {boolean} cfg.hidden flag to show/hide the menu
+  * @prop {object[]} cfg.items this property contains the items injected from the other plugins
+  * @prop {boolean} cfg.isAdmin flag to show/hide the admin menu
+  * @prop {string} cfg.displayName the name of the user to display
+  * @prop {boolean} cfg.showAccountInfo flag to show/hide the account info
+  * @prop {string} cfg.bsStyle the style of the button
+  * @prop {string} cfg.className the class name of the button
+  * @prop {object} context.loadedPlugins the plugins loaded in the application
```

Comment:
My suggestion is to add only cfg properties and item exposed and configurable by the localConfig.json.

We should **not** add inside plugin documentation:
- props connected to the state (e.g.: enableRulesManager, enableImporter, user, isAdmin)
- props function connected to an action (e.g.: onItemSelected). In general functions cannot be easily used and configured in the localConfig.json file
- context argument

For the moment let's keep only the already documented ones in [ManagerMenu](https://dev-mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.ManagerMenu) and [Login](https://dev-mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.Login) so: `cfg.id`, `cfg.isUsingLDAP` and `items`

Also items should be placed before it's own description about containers usage

```suggestion
  * @prop {boolean} cfg.isUsingLDAP flag refers to if the user with type LDAP or not to manage show/hide change psasword, by default: false
  * @prop {object[]} items this property contains the items injected from the other plugins
```

Reply:
Done